soh/soh/SaveManager.cpp

```diff
@@ -139,6 +139,16 @@ void SaveManager::LoadRandomizerVersion2() {
         });
     });
 
+    SaveManager::Instance->LoadArray("entrances", ARRAY_COUNT(gSaveContext.entranceOverrides), [&](size_t i) {
```

Comment:
This will only contain shuffled entrances as it is coded now (both directions, so when decoupled entrances == ON would consume the same amount of indexes in this array as == OFF)